src/redux/books/booksSlice.js

```diff
@@ -1,22 +1,40 @@
 import { createSlice } from '@reduxjs/toolkit';
 
-const initialState = {
-  books: [],
-};
+const initialState = [
+  {
+    item_id: 'item1',
+    title: 'The Great Gatsby',
+    author: 'John Smith',
+    category: 'Fiction',
+  },
+  {
+    item_id: 'item2',
+    title: 'Anna Karenina',
+    author: 'Leo Tolstoy',
+    category: 'Fiction',
+  },
+  {
+    item_id: 'item3',
+    title: 'The Selfish Gene',
+    author: 'Richard Dawkins',
+    category: 'Nonfiction',
+  },
+];
```

Comment:
- [ ] Everything works fine with regards to books :clap: But for categories: when clicking the button the message should show, kindly fix it

Reply:
Hello @GzimAsani would you please show me in what part of the project [requirements](https://github.com/microverseinc/curriculum-react-redux/blob/main/bookstore/use_redux_in_react_components_v2.md) says that part about the categories?
 ## Project requirements
 - Wrap your <App> with the <Provider> component from react-redux
- Pass your configured store into the <Provider>
- Create an initialState variable for your booksSlice, which will include the following:
`
// Initial state:
[
  {
    "item_id": "item1",
    "title": "The Great Gatsby",
    "author": "John Smith",
    "category": "Fiction"
  },
  {
    "item_id": "item2",
    "title": "Anna Karenina",
    "author": "Leo Tolstoy",
    "category": "Fiction"
  },
  {
    "item_id": "item3",
    "title": "The Selfish Gene",
    "author": "Richard Dawkins",
    "category": "Nonfiction"
  }
]
`
- Display your books, received from the slice, in a reusable component
- Dispatch actions using useDispatch
- Add a <Button> component, which includes:
- An event handler that adds a book to the books array (with attributes id, title and author)
- Add a <Button> component, which includes:
- An event handler that removes a book from the books array (by id)